cypress/e2e/SiteAlert/StandardAlert.feature

```diff
@@ -0,0 +1,159 @@
+            | id="site-alert-example6"                                                           |
+            | data-site-alert-closable="true"                                                    |
+            | >                                                                                  |
+            | <div class="usa-alert">                                                            |
+            | <div class="usa-alert__body">                                                      |
+            | <div class="usa-alert__nci-header">                                                |
+            | <h3 class="usa-alert__heading">COVID-19 resources.</h3>                            |
+            | </div>                                                                             |
+            | <div class="usa-alert__nci-content" id="site-alert-content3">                      |
+            | <ul class="usa-alert__nci-list">                                                   |
+            | <li>                                                                               |
+            | <a class="usa-link" href="#">What people with cancer should know</a>               |
+            | </li>                                                                              |
+            | <li>                                                                               |
+            | <a class="usa-link" href="#">Get the latest public health information from CDC</a> |
+            | </li>                                                                              |
+            | </ul>                                                                              |
+            | </div>                                                                             |
+            | </div>                                                                             |
+            | </div>                                                                             |
+            | </section>                                                                         |
+        And user hits "Save" button
+        When user clicks "Block layout" tab
+        And user places the "Site Alert Area" block
+        And user filters by "NCIDS Alert"
+        And user unchecks show title and clicks save
+
+
+    Scenario: Standard site alert
+        Given user is navigating to "/"
+        Then standard site alert is displayed
+        And alert has "Dismiss alert" button
+        When user clicks on "expand" arrow button
+        Then alert is "expanded"
+        And the following links are exposed
+            | linkText                                          | href |
+            | What people with cancer should know               | #    |
+            | Get the latest public health information from CDC | #    |
+        When user clicks on "collapse" arrow button
+        Then alert is "collapsed"
+        And alert links are hidden
```

Comment:
Wait -- the alert defaults to being collapsed.

Can you please add a test for expanding then moving to another page and seeing it is still expanded? Then collapse it and move to another page and seeing it is still collapsed.

Reply:
added steps to verify alert state